.github/workflows/publish-on-comment.yaml

```diff
@@ -0,0 +1,78 @@
+name: publish-on-comment
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  check-comment:
+    name: Check comment for /publish
+    runs-on: ubuntu-latest
+    if: ${{ github.event.issue.pull_request }}
+    outputs:
+      trigger-comment: ${{ steps.check.outputs.triggered }}
+    steps:
+      - uses: khan/pull-request-comment-trigger@v1.1.0
+        id: check
+        with:
+          trigger: "/publish"
+          prefix_only: "true"
+          reaction: "rocket"
+
+  get-affected-plugins:
+    name: Get affected plugins
+    needs: check-comment
+    if: ${{ needs.check-comment.outputs.trigger-comment == 'true' }}
+    uses: ./.github/workflows/nx.template.yaml
+    with:
+      nx-head: ${{ github.head_ref && format('refs/pull/{0}/merge', github.event.number) || github.ref_name  }}
```

Comment:
`github.event.number` doesn't exists on event `issue_comment `

Reply:
for reference

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#issue_comment

https://docs.github.com/en/actions/learn-github-actions/contexts#github-context

https://docs.github.com/en/webhooks/webhook-events-and-payloads#issue_comment